templates/static_templates/theme-acc.html

```diff
@@ -0,0 +1,30 @@
+<%! from django.utils.translation import ugettext as _ %>
```

Comment:
Are you sure this file will work with the theme? I see you have the same contents inside `theme-honor.html`. I'd suggest to keep it in one place only.